test/functional/wallet_importmulti.py

```diff
@@ -119,9 +126,13 @@ def get_multisig(self):
                         CScript([OP_HASH160, witness_script, OP_EQUAL]).hex(),  # p2sh-p2wsh
                         script_to_p2sh_p2wsh(script_code))  # p2sh-p2wsh addr
 
-    def test_importmulti(self, req, success, error_code=None, error_message=None):
+    def test_importmulti(self, req, success, error_code=None, error_message=None, warnings=[]):
         """Run importmulti and assert success"""
         result = self.nodes[1].importmulti([req])
+        observed_warnings = []
+        if 'warnings' in result[0]:
+           observed_warnings = result[0]['warnings']
```

Comment:
Indentation is not a multiple of four :-)

Reply:
Upstream